tests/e2e/e2e_test.go

```diff
@@ -31,33 +35,119 @@ var (
 
 	installCTK bool
 
-	image string
+	ImageRepo string
+	ImageTag  string
 
-	sshKey  string
-	sshUser string
-	host    string
-	sshPort string
-)
+	sshKey      string
+	sshUser     string
+	host        string
```

Comment:
Out of scope for this PR, but we could consider using `sshHost` here instead for consistency.

Reply:
Done